ui_framework/doc_site/src/views/popover/popover_anchor_position.js

```diff
@@ -43,38 +47,339 @@ export default class extends Component {
     });
   }
 
+  onButtonClick3() {
```

Comment:
@cjcenizal This is terrible. I was copying your pattern, but I'm sure I can do this dynamically. If you happen to have a snippet though I'll take it!